src/Telegram.Bot/ITelegramBotClient.cs

```diff
@@ -578,21 +578,39 @@ public interface ITelegramBotClient
             string fileId,
             CancellationToken cancellationToken = default);
 
+        /// <summary>
+        /// Use this method to get download a file. Get <paramref name="filePath"/> by calling <see cref="GetFileAsync"/>
```

Comment:
Should be `Use this method to download a file.`

Reply:
Fixed